tests/test_bucketlist_resource.py

```diff
@@ -76,7 +81,61 @@ def test_cant_view_bucketlists_without_token(self):
         self.assertTrue(response.status_code == 401)
         self.assertTrue(b'Authorization Required' in response.data)
 
-    #view specific bucketlist
+    def test_user_cant_view_second_bucketlist_when_limit_is_one(self):
+        self.headers["Content-Type"] = "None"
```

Comment:
Why did you set Content-Type to None? I think this might interfere with what you are trying to test.